content/docs/get-started/azure/review-project.md

```diff
@@ -110,16 +110,31 @@ This Pulumi program creates an Azure resource group and storage account and expo
 **Note**: In this program, the location of the resource group is set in the configuration setting `azure:location` (check the `Pulumi.dev.yaml` file). This is an easy way to set a global location for your program so you don't have to specify the location for each resource manually. The location for the storage account is automatically derived from the location of the resource group. To override the location for a resource, simply set the location property to one of Azure's [supported locations](https://azure.microsoft.com/en-us/global-infrastructure/locations/).
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python -m venv azure-env
```

Comment:
I believe our offline discussion was that we in general cannot expect `Python` to be Python 3?

Cc also @jen20.

Reply:
For Linux, CentOS doesn't have a default python, Ubuntu/Debian includes 2 and 3. macOS before Catalina has 2.7 as default and python3 has to be user installed. In those cases, `python3` would be correct. Let's go with `python3`  for now until version checking for python is implemented.